scripts/tasks/0_init_config.ipynb

```diff
@@ -0,0 +1,73 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    from pathlib import Path
</pre><p>Using <code>pathlib.Path</code> instead of <code>os.path.join</code> to handle system Paths.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mscheltienne/eeg-flow/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>